docs_src/pipelinebenchmarking.md

```diff
@@ -144,24 +143,19 @@ sudo ./benchmark.sh --stream_density <target FPS> --logdir <output dir>/data --i
 
 ---
 
-#### Workload
-We are currently supporting 2 types of workloads:
-    1. dlstreamer
-    2. ovms
-
-These are the input value for `--workload` parameter for benchmark.sh script. The default value for `--workload` parameter is `dlstreamer` in case it is not provided when running benchmark.sh script.
 
 #### Benchmark Specified Profile for OVMS
-For running ovms workload, we are supporting different programming languages and different models. You may specify [language choice](./OVMS/supportingDifferentLanguage.md) and [model input](./OVMS/supportingDifferentModel.md). Then you may **prefix** benchmark script run command with specific profile.
+
+There are several pipeline profiles for the OVMS workload.  With the current existing OVMS profiles, we are supporting different programming languages and different pipeline models. You may specify [language choice](./OVMS/supportingDifferentLanguage.md) and [model input](./OVMS/supportingDifferentModel.md). Then you may **prefix** benchmark script run command with specific profile.
```

Comment:
can reword to make it a bit more generic. "There are several pipeline profiles that support different programming languages and inference models."

Reply:
fixed